Author: aszlig <aszlig@nix.build>
Date:   Tue Mar 27 05:37:02 2018 +0200

    Remove unneeded dependencies
    
    These dependencies are mostly for external services we certainly do not
    need.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: strip-dependencies.patch

diff --git a/package.json b/package.json
index f197386f4..a292c772c 100644
--- a/package.json
+++ b/package.json
@@ -4,13 +4,9 @@
   "version": "4.44.2",
   "main": "./website/server/index.js",
   "dependencies": {
-    "@slack/client": "^3.8.1",
     "accepts": "^1.3.5",
-    "amazon-payments": "^0.2.7",
-    "amplitude": "^3.5.0",
     "apidoc": "^0.17.5",
     "autoprefixer": "^8.5.0",
-    "aws-sdk": "^2.239.1",
     "axios": "^0.18.0",
     "axios-progress-bar": "^1.2.0",
     "babel-core": "^6.26.3",
@@ -49,10 +45,8 @@
     "gulp-nodemon": "^2.2.1",
     "gulp.spritesmith": "^6.9.0",
     "habitica-markdown": "^1.3.0",
-    "hellojs": "^1.15.1",
     "html-webpack-plugin": "^3.2.0",
     "image-size": "^0.6.2",
-    "in-app-purchase": "^1.9.4",
     "intro.js": "^2.9.3",
     "jquery": ">=3.0.0",
     "js2xmlparser": "^3.0.0",
@@ -64,32 +58,23 @@
     "mongoose": "^5.1.2",
     "morgan": "^1.7.0",
     "nconf": "^0.10.0",
-    "node-gcm": "^0.14.4",
     "node-sass": "^4.9.0",
     "nodemailer": "^4.6.4",
     "ora": "^2.1.0",
     "pageres": "^4.1.1",
     "passport": "^0.4.0",
-    "passport-facebook": "^2.0.0",
-    "passport-google-oauth20": "1.0.0",
-    "paypal-ipn": "3.0.0",
-    "paypal-rest-sdk": "^1.8.1",
     "popper.js": "^1.14.3",
     "postcss-easy-import": "^3.0.0",
     "ps-tree": "^1.0.0",
     "pug": "^2.0.3",
-    "push-notify": "git://github.com/habitrpg/push-notify.git#6bc2b5fdb1bdc9649b9ec1964d79ca50187fc8a9",
-    "pusher": "^1.3.0",
     "rimraf": "^2.4.3",
     "sass-loader": "^7.0.0",
     "shelljs": "^0.8.2",
-    "stripe": "^5.9.0",
     "superagent": "^3.8.3",
     "svg-inline-loader": "^0.8.0",
     "svg-url-loader": "^2.3.2",
     "svgo": "^1.0.5",
     "svgo-loader": "^2.1.0",
-    "universal-analytics": "^0.4.16",
     "url-loader": "^1.0.0",
     "useragent": "^2.1.9",
     "uuid": "^3.0.1",
@@ -106,7 +91,6 @@
     "webpack": "^3.12.0",
     "webpack-merge": "^4.0.0",
     "winston": "^2.4.2",
-    "winston-loggly-bulk": "^2.0.2",
     "xml2js": "^0.4.4"
   },
   "private": true,
@@ -186,7 +170,6 @@
     "webpack-hot-middleware": "^2.22.2"
   },
   "optionalDependencies": {
-    "memwatch-next": "^0.3.0",
-    "node-rdkafka": "^2.3.0"
+    "memwatch-next": "^0.3.0"
   }
 }
